src/providers/fault-store/fault-store.ts

```diff
@@ -88,7 +89,7 @@ export class FaultStoreProvider {
    *
    * {@link TestSummaryComponent}
    */
-  getTestResultAndCalculateFaultTotals(): TestResult {
+  getTestResultAndCalculateFaultTotals(): void {
```

Comment:
hym, can we rename this method? it has the prefix 'get' but do not returning anything